test/internet/test-dgram-broadcast-multi-process.js

```diff
@@ -153,7 +153,8 @@ if (process.argv[2] !== 'child') {
               assert.strictEqual(
                 count,
                 messages.length,
-                'A worker received an invalid multicast message'
+                `A worker received an invalid multicast message.
+                Received ${messages.length} and wanted ${count}.`
```

Comment:
This message includes extra spaces.

Reply:
This happens because it is a multi line template strings. In this case all spaces in the beginning of the second line are real spaces in the string. In Node.js we limit every string to a single line to prevent things like that. It should therefore either be changed now or while landing.

Reply:
@nikkistonge It would be acceptable to use concatenation here:

```js
    'A worker received an invalid multicast message.' +
    `Received ${messages.length} and expected {$count}.`
```

Reply:
Sorry about the delay, didn't see these comments.

I didn't know that was how template strings worked, I've only ever used them for SQL and html stuff and both don't care about extra spaces so I assumed they got removed.

Changed my code to remove them and added just one space after the first sentence. @Trott